src/test/persistent/file_processor_test.go

```diff
@@ -0,0 +1,49 @@
+package persistent_test
+
+import (
+	"os"
+	"testing"
+
+	"ses.genesis.com/exchange-web-service/src/main/persistent"
+)
+
+var testFileProcessor = persistent.NewFileProcessor("test_file_storage.txt")
```

Comment:
Можливо `test_file_storage.txt` можна винести як константу
```suggestion
var testFileProcessor = persistent.NewFileProcessor("test_file_storage.txt")
```

Reply:
Можливо, але для тестів як на мене краще мати значення "на виду"